registrations/utils.py

```diff
@@ -87,6 +91,43 @@ def has_allowed_substitute_user_email_domain(email_address):
     )
 
 
+def create_event_ics_file_content(event, language="fi"):
+    cal = Calendar()
+    # Some properties are required to be compliant
+    cal.add("prodid", "-//linkedevents.hel.fi//NONSGML API//EN")
+    cal.add("version", "2.0")
+
+    with translation.override(language):
+        calendar_event = CalendarEvent()
+
+        if (start_time := event.start_time) and (name := event.name):
+            calendar_event.add("dtstart", start_time)
+            calendar_event.add("summary", name)
+        else:
+            raise ValueError(
+                "Event doesn't have start_time or name. Ics file cannot be created."
+            )
+
+        calendar_event.add("dtend", (event.end_time) if event.end_time else start_time)
```

Comment:
```suggestion
        calendar_event.add("dtend", event.end_time if event.end_time else start_time)
```
Parenthesis are not commonly used in Python in these kind of expressions.

Reply:
I actually forgot to remove those parenthesis. Changed as suggested